src/components/Timer/Timer.js

```diff
@@ -1,40 +1,47 @@
 import React, { useState, useEffect } from 'react';
 
 import styled from 'styled-components';
-import {post} from '../../requests';
+import { post } from '../../requests';
 
 const Timer = () => {
-    const SPACE = 32
-    const [time, setTime] = useState("00:00:00")
+    const SPACE = 32;
+    const countdownInMills = 15000;
+    const [time, setTime] = useState("00:15:00");
     const [running, setRunning] = useState(false);
```

Comment:
Let's rename it to `[isRunning, setIsRunning]`  and stick to this notation in future - every boolean-logic variable should be prefixed with `is`.